espnet2/text/korean_cleaner.py

```diff
@@ -0,0 +1,77 @@
+
+## Referenced from https://github.com/hccho2/Tacotron-Wavenet-Vocoder-Korean
+
+import re
+
+class Korean_cleaner:
```

Comment:
I like CamelCase for class name.
```suggestion
class KoreanCleaner:
```